packages/teams-js/CHANGELOG.md

```diff
@@ -1,9 +1,29 @@
 # Change Log - @microsoft/teams-js
 
-This log was last generated on Wed, 02 Aug 2023 14:49:19 GMT and should not be manually modified.
+This log was last generated on Wed, 06 Sep 2023 20:30:49 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## 2.15.0
+
+Wed, 06 Sep 2023 20:30:49 GMT
+
+### Minor changes
+
+- Removed `appNotification` capability and collateral
+- Added `clipboard` capability, allowing access to the system clipboard programmatically
+- Fixed bugs within the `appNotification` capability and its validation
+- Add MacOS support
```

Comment:
Add**ed** MacOS support

Reply:
good catch, seems this passed through. I'll manually update the file